src/types/big-integer.ts

```diff
@@ -9,13 +9,29 @@ export const types = {
 	},
 };
 
-export type Types = TypeUtils.TsTypes<number, number>;
-type DbWriteType = number;
+export type Types = TypeUtils.TsTypes<bigint, bigint>;
```

Comment:
I think the write type could keep the number as `number | bigint` since it's valid to pass in either - there's no need to use a bigint if a number is fine for the specific use

Reply:
Good point, I probably just blindly replaced `number` with `bigint` here